src/helix/mod.rs

```diff
@@ -140,9 +140,9 @@
 //!
 //! | Endpoint | Helper | Module |
 //! |---|---|---|
-//! | [Get Conduits](https://dev.twitch.tv/docs/api/reference#get-conduits) | [`HelixClient::get_conduits`] | [`eventsub::get_conduits`] |
+//! | [Get Conduits](https://dev.twitch.tv/docs/api/reference#get-conduits) | [`HelixClient::get_conduit_shards`], [`HelixClient::get_conduits`] | [`eventsub::get_conduits`] |
```

Comment:
this is wrong, not sure how to best avoid this

Reply:
(Not sure if that works): We could go through the methods twice. First, search for exact matches and mark these as "done" (or remove them). And second, go through the remaining ones and find prefix matches.

Reply:
I've attempted that in https://github.com/twitch-rs/twitch_api/pull/443/commits/5bc7be817a935909b8aa15e32690c40161dd7fb6